### PR TITLE
New packages: Pforth, Gforth, chez-scheme, LuaJIT, retro

### PR DESCRIPTION
**Pull request by ghost**
Please fill out the template.

Read: https://k1ss.org/wiki/kiss/style-guide

---

## Description of package
LuaJIT: A Just-In-Time compiler for the Lua programming language.
chez-scheme: A powerful implementation of the Scheme programming language.
PForth: A portable implementation of the Forth programming language.
GForth: An implementation of the ANS standard for the Forth programming language.

## Installed manifest of package
LuaJIT: https://termbin.com/hkkm
chez-scheme: https://termbin.com/vi63
pforth: https://termbin.com/i2im
gforth: https://termbin.com/s47g
retro: https://termbin.com/wpu1

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @mmatongo if I can no longer maintain this package.

### CODE REVIEW

File: languages/gforth/build [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+CFLAGS="$CFLAGS -std=gnu99" ./configure \
+    --prefix=/usr \
+    --without-check
+
+make -j1
```
Comment (mmatongo) [inline languages/gforth/build:7]: Can we use `make -j$(nproc)` ?

**Review by mmatongo — APPROVED** [review]

File: languages/luajit/depends [hunk 2]
```diff
@@ -0,0 +1 @@
+gcc
```
Comment (mmatongo) [inline languages/luajit/depends]: `gcc` is implicit not so, I think we can do without the depends file for `luajit`.

**Review by mmatongo — APPROVED** [review]

**Review by mmatongo — APPROVED** [review]

**Review by mmatongo — APPROVED** [review]

**Review by mmatongo — APPROVED** [review]

**Review by mmatongo — APPROVED** [review]

File: languages/luajit/depends (hunk 2 above)
Comment (ghost) [reply languages/luajit/depends]: It depends on libgcc, which is why I listed it.

File: languages/gforth/build (hunk 1 above)
Comment (ghost) [reply languages/gforth/build:7]: Sadly not. Parallel builds cause issues, at least in my tries.

File: languages/luajit/depends (hunk 2 above)
Comment (ghost) [reply languages/luajit/depends]: I'll remove it, however.

File: languages/gforth/build (hunk 1 above)
Comment (mmatongo) [reply languages/gforth/build:7]: Alright

File: languages/luajit/depends (hunk 2 above)
Comment (mmatongo) [reply languages/luajit/depends]: Awesome

**Review by mmatongo — APPROVED** [review]

**Pull request merged**